### PR TITLE
feat(balance): Buffed survivor armors, Plastic Armors

### PR DESCRIPTION
**Pull request by yay855**
Made the few survivor armors I missed last time I did this have armor values close to the kevlar vests they use as ingredients. Also decreased football armor and hard plastic chestguards' encumbrance, they were encumbering as hell despite being half-decent earlygame armor.

### Required

- [x] I wrote the PR title in [conventional commit format](https://docs.cataclysmbn.org/en/contribute/changelog_guidelines/).
- [x] I ran the [code formatter](https://docs.cataclysmbn.org/en/contribute/contributing/#code-style).
- [ ] I linked any relevant issues using [github keyword syntax](https://docs.cataclysmbn.org/en/contribute/contributing/#pull-request-notes) so it can be closed automatically.
- [x] I have [committed my changes to new branch that isn't `main`](https://docs.cataclysmbn.org/en/contribute/contributing/#make-your-changes) so it won't cause conflict when updating `main` branch later.

## Purpose of change

There were multiple survivor armor pieces that had much worse protection than the kevlar vest they used as a crafting ingredient, and football armor and hard chestguards were way too encumbering for how mediocre they are as armor.

## Describe the solution

The survivor duster, survivor trenchcoat, scavenger armor, and corresponding gear pieces (boots, gloves, hoods, etc) have had their thickness increased in order to boost their armor values. Scavenger gear has had its recipe and description changed to be less dependent on pre-cataclysm equipment, since that's kind of an odd concept to begin with at such high tailoring levels.

The football armor and hard chestguard items have had their encumbrance lowered. They are early to midgame armor pieces, they do not need to have 20-35 encumbrance.

### CODE REVIEW

**Review by chaosvolt — COMMENTED** [review]
I should point out that you're removing submaterials from a lot of survivor items. I feel like this is a bad idea because it does two things in particular:
1. Makes the items a lot more bland and same-y in terms of protection stats.
2. Makes them more expensive to repair.

This was already done to the standard tiers of survivor suit for example, so this will make the affected items just a bit less distinct from them.

**Comment by yay855** [comment]
> I should point out that you're removing submaterials from a lot of survivor items. I feel like this is a bad idea because it does two things in particular:
> 
>     1. Makes the items a lot more bland and same-y in terms of protection stats.
> 
>     2. Makes them more expensive to repair.
> 
> 
> This was already done to the standard tiers of survivor suit for example, so this will make the affected items just a bit less distinct from them.

Right, that's reasonable. I can increase the thickness instead, I suppose.

**Comment by chaosvolt** [comment]
Either could work, ye

**Comment by Coolthulhu** [comment]
How does it change the actual armor values?

**Comment by ZChris13** [comment]
the survivor duster niche exists entirely as a lightly encumbering armor that grants wet protection that won't get torn up by everything you see, in comparison to the rain coat and unfolded poncho
I don't think it needs to be super bulletproof

**Comment by RoyalFox2140** [comment]
> the survivor duster niche exists entirely as a lightly encumbering armor that grants wet protection that won't get torn up by everything you see, in comparison to the rain coat and unfolded poncho I don't think it needs to be super bulletproof

It should be as protective as the kevlar vest used to craft it at a bare minimum or you're in a situation where reality has broken down.

**Comment by chaosvolt** [comment]
> It should be as protective as the kevlar vest used to craft it at a bare minimum or you're in a situation where reality has broken down.

I mean, it depends. If you're spreading the kevlar vest out over a larger garment, then that means a lower layer count for any given area. Otherwise we'd have to set it so either only the torso gets max protection, or you have to use more than one, or use more than one vest to line all parts of the item with the same thickness of kevlar.

**Comment by ZChris13** [comment]
If the duster was supposed to be armor it wouldn't have only 85% coverage.

**Comment by yay855** [comment]
Maybe reflavoring the survivor duster into a lightly kevlar lined duster? That could work. Instead of taking a full kevlar vest, just take a few pieces of kevlar.

File: data/json/items/armor/torso_armor.json [hunk 1]
```diff
@@ -331,14 +331,14 @@
     "volume": "6 L",
     "price": "600 USD",
     "price_postapoc": "40 USD",
-    "material": [ "kevlar", "cotton" ],
+    "material": [ "kevlar" ],
```
Comment (chaosvolt) [inline data/json/items/armor/torso_armor.json:334]:
```suggestion
    "material": [ "kevlar", "cotton" ],
```

File: data/json/items/armor/torso_armor.json [hunk 2]
```diff
@@ -331,14 +331,14 @@
...
     "symbol": "[",
     "looks_like": "lsurvivor_suit",
     "color": "green",
     "covers": [ "torso" ],
     "coverage": 95,
     "encumbrance": 16,
-    "storage": "2500 ml",
+    "storage": "3500 ml",
     "warmth": 15,
     "material_thickness": 4,
```
Comment (chaosvolt) [inline data/json/items/armor/torso_armor.json:343]:
```suggestion
    "material_thickness": 5,
```

**Review by chaosvolt — COMMENTED** [review]
Will make light survivor body armor consistent with survivor cargo pants.

**Comment by chaosvolt** [comment]
> How does it change the actual armor values?

Survivor wetsuit:
image

Survivor firesuit:
image

Survivor duster:
image

Survivor cargo pants (light survivor body armor matches this value):
image

Light survivor cargo pants:
image

Scavenger cowl:
image

Scavenger gear:
image

Side note: I just realized light survivor body armor maybe could stay the same strength as light survivor cargo pants, but the fact we have two different strengths of survivor cargo pants is weird.

**Comment by chaosvolt** [comment]
General thoughts:
1. Scavenger gear and cowl have pretty damn high protection values. Then again, they don't really have any niche in between nomad gear and actual survivor gear anyway that comes to mind so them being beefed up doesn't really screw with balance.
2. Survivor cargo pants and light survivor cargo pants are just...weird. It makes it more of a headache for me to figure out which one the light survivor body armor is supposed to be consistent with since that item doesn't have two separate tiers like the pants do.

Other than those stray thoughts the buffs are okay I guess. All the survivor clothing still stays below light survivor suits protection-wise which is where they should be. Scav gear beats it but again scavenger gear doesn't really fit in anywhere so doesn't matter.

**Comment by yay855** [comment]
TBH we might be better off just obsoleting the light survivor cargo pants.

**Review by chaosvolt — APPROVED** [review]

**Pull request merged**